psychopy/data/trial.py

```diff
@@ -420,10 +420,12 @@ def _createOutputArray(self, stimOut, dataOut, delim=None,
             for thisDataOut in dataOut:
                 # make a string version of the data and then format it
                 tmpData = dataAnal[thisDataOut][stimN]
+                replaceNone = False
                 if hasattr(tmpData, 'tolist'):  # is a numpy array
                     strVersion = str(tmpData.tolist())
                     # for numeric data replace None with a blank cell
-                    if tmpData.dtype.kind not in ['SaUV']:
+                    if tmpData.dtype.kind not in 'SaUV':
```

Comment:
What's the motivation for removing the []? Is it possible for `.dtype.kind` to be just part of the string `"SaUV"`, and if so do we want to catch it here?

Reply:
Yeah I don't think `'SaUV'` itself is a valid `dtype.kind`. I think it has always meant to check whether `.dtype.kind` is part of the string, and it was a typo to include the `[]`. Right now even for string data, `'None'` gets replaced by `''` because `tmpData.dtype.kind` is never the same as `'SaUV'`.

Reply:
Ah okay, I don't know much about dtypes tbh so I'll take your word on it. To be clearest we should probably do `("S", "a", "U", "V")`, but this is a minor point, the string is fine :P

Reply:
Might be worth checking with Jon or whoever wrote it in the first place? I don't think `"a"` is one of the valid character codes anymore: https://numpy.org/doc/stable/reference/generated/numpy.dtype.kind.html